packages/driver-interface/src/botRoutes.ts

```diff
@@ -70,24 +39,15 @@ export const init = (bot: Telegraf<TelegrafContext>): void => {
 
   bot.on('message', (ctx) => {
     const msg = ctx.message
-
-    /** Login attempt from command /login. */
-    if (msg.text && msg.text.toLowerCase().includes('pmv-')) {
-      botServices.onLogin(msg.text, ctx)
-    }
-
-    if (!msg.location) return
-
-    return botServices.onLocationMessage(msg, ctx)
+    if (msg.contact && msg.contact.phone_number)
```

Comment:
Now, transport is highely tied to user phone and wont work from any other phone. Do we want that? If driver is sick and replaced etc.?

Reply:
I think that's fine for now and should be modified in another iterator if it poses a problem